bitmap-plasma/app/build.gradle

```diff
@@ -19,7 +19,7 @@ model {
 
     android.buildTypes {
         release {
-            isMinifyEnabled = false
+            minifyEnabled = false
```

Comment:
The debug build is now debuggable by default.  I don't think it would work if you are trying to debug the release build.